README.md

```diff
@@ -387,12 +410,29 @@ resources:
 - # (your resources)
 
 # Add these lines to your kustomization.yaml:
-commonAnnotations: 
+commonAnnotations:
   argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
 ```
 
 See the Argo CD docs [for more on this sync option](https://argo-cd.readthedocs.io/en/stable/user-guide/sync-options/#skip-dry-run-for-new-custom-resources-types). See the [redhat-cop/gitops-catalog](https://github.com/redhat-cop/gitops-catalog) for examples of this option used with Kustomize and OLM-installed operators.
 
+### Q: Logging into ArgoCD stops working and shows message: Failed to query provider.
+If you stopped and started your cluster, a timing issue might cause the following message to appear
+when you try logging into ArgoCD with `Log In Via OpenShift`:
+```
+Failed to query provider "https://openshift-gitops-server-openshift-gitops.apps.myserver.mydomain.com/api/dex": Get "http://openshift-gitops-dex-server.openshift-gitops.svc.cluster.local:5556/api/dex/.well-known/openid-configuration": dial tcp 1##.###.###.###:5556: connect: connection refused
+```
+To correct this problem:
++ List the pods:
```

Comment:
`oc delete pods -n openshift-gitops -l app.kubernetes.io/name=openshift-gitops-dex-server` -> one liner to delete correct pod.

Reply:
changed